Makefile

```diff
@@ -88,7 +88,7 @@ libutf8proc.so: libutf8proc.so.$(MAJOR).$(MINOR).$(PATCH)
 	ln -f -s libutf8proc.so.$(MAJOR).$(MINOR).$(PATCH) $@
 	ln -f -s libutf8proc.so.$(MAJOR).$(MINOR).$(PATCH) $@.$(MAJOR)
 
-libutf8proc.$(MAJOR).dylib: utf8proc.o
+libutf8proc.$(MAJOR).dylib:
```

Comment:
This dependency is correct: the library is built from utf8proc.o. `$^` refers to this.